TPP.Persistence/Repos/IUserRepo.cs

```diff
@@ -10,6 +11,7 @@ public interface IUserRepo
         public Task<User> RecordUser(UserInfo userInfo);
         public Task<User?> FindBySimpleName(string simpleName);
         public Task<User?> FindByDisplayName(string displayName);
+        public Task<List<User>> ListAllUnderPokeyen(long yen);
```

Comment:
the method name seems odd compared to the rest. suggestion: `FindByPokeyenLessThan`

Reply:
I originally named it like this to make it clear that it returns a list rather than a single user, however any IDE will tell you that anyway so I guess its rather unnecessary.

Reply:
I believe "FindAll" is a good convention for this. That's how I've seen it in other ORMs at least